nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -368,6 +369,11 @@ def decode_hypothesis(self, hypotheses_list: List[Hypothesis]) -> List[Union[Hyp
             # De-tokenize the integer tokens
             hypotheses_list[ind].text = hypothesis
 
+            if self.compute_langs:
+                hypotheses_list[ind].langs = self.decode_tokens_to_lang(prediction)
```

Comment:
Doesn't this only work if the Tokenizer is an aggregate one ? Please check it in the constructor and raise an error.

Reply:
fixed